src/Playwright.CLI/Program.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Playwright.CLI
+{
+    class Program
+    {
+        private const string DriverEnvironmentPath = "PW_CLI_DRIVERPATH";
+
+        static void Main(string[] args)
+        {
+            string pwPath = GetFullPath();
+            var playwrightStartInfo = new ProcessStartInfo(pwPath, string.Join(' ', args))
+            {
+                UseShellExecute = false,
+                CreateNoWindow = true,
+                RedirectStandardError = true,
+                RedirectStandardInput = true,
+                RedirectStandardOutput = true
+            };
+
+            var pwProcess = new Process()
+            {
+                StartInfo = playwrightStartInfo,
+            };
+
+            playwrightStartInfo.EnvironmentVariables.Add("PW_CLI_TARGET_LANG", "csharp");
+            pwProcess.Start();
+
+            pwProcess.WaitForExit();
+            Console.WriteLine(pwProcess.StandardOutput.ReadToEnd());
```

Comment:
I think we should inherit stdio streams to get incremental input/output/errors, instead of writing something at the end.

Reply:
Do we need that having `RedirectStandardOutput` set to `true`?

Reply:
@kblok the default is `false` - so I think we do, if we want to see the input/output

Reply:
sorry. I mean, if we have `RedirectStandardOutput = true` do we need to write the output to the console. Doesn't `RedirectStandardOutput` do that for us?

Reply:
So, I assumed that too, but it didn't. I'll give it another go, though.

Reply:
Heh, look at that, turns out you have to call `BeginOutputReadLine`, to get anything on the event handlers :D

Reply:
This is done, now. For now, I'm _assuming_ that what I get is a direct `WriteLine`, so I don't support some scenarios, like the app clearing the output. But I think this will be fine for v1.